Exporter fails with "inconsistent label cardinality" for gluster_volume_counts

## Symptom

When the collector `gluster_volume_counts` is enabled, the exporter fails to start.

## Configuration

```
[globals]
gluster-cluster-id = ""
gluster-mgmt = "glusterd"
glusterd-dir = "/var/lib/glusterd"
gluster-binary-path = "gluster"
port = 8090
metrics-path = "/metrics"
log-dir = "/var/log/gluster-exporter"
log-file = "exporter.log"
log-level = "debug"

[collectors.gluster_volume_counts]
name = "gluster_volume_counts"
sync-interval = 5
disabled = false
```

## Versions

```
root@node-1:~# cat /etc/debian_version 
10.0

root@node-1:~# gluster --version
glusterfs 6.1
Repository revision: git://git.gluster.org/glusterfs.git
Copyright (c) 2006-2016 Red Hat, Inc. <https://www.gluster.org/>
GlusterFS comes with ABSOLUTELY NO WARRANTY.
It is licensed to you under your choice of the GNU Lesser
General Public License, version 3 or any later version (LGPLv3
or later), or the GNU General Public License, version 2 (GPLv2),
in all cases as published by the Free Software Foundation.
```


Exporter built from https://github.com/gluster/gluster-prometheus/commit/4baab01ff49929ec518eacf7e0faa0dedbf06770
With the GD1 remote-host workaround described in #151 

## Logs

```
root@node-1:~# gluster-exporter --config=/etc/gluster-exporter/gluster-exporter.toml
panic: inconsistent label cardinality

goroutine 73 [running]:
github.com/gluster/gluster-prometheus/vendor/github.com/prometheus/client_golang/prometheus.(*MetricVec).With(0xc000062ec0, 0xc0002712e8, 0xc00030cb00, 0xa8bdc0)
	/opt/gluster-exporter/src/github.com/gluster/gluster-prometheus/vendor/github.com/prometheus/client_golang/prometheus/vec.go:146 +0x79
github.com/gluster/gluster-prometheus/vendor/github.com/prometheus/client_golang/prometheus.(*GaugeVec).With(0xc00000e160, 0xc0002712e8, 0x9e8741, 0xa)
	/opt/gluster-exporter/src/github.com/gluster/gluster-prometheus/vendor/github.com/prometheus/client_golang/prometheus/gauge.go:115 +0x38
main.volumeCounts(0xa8d920, 0xc000063940, 0x0, 0x0)
	/opt/gluster-exporter/src/github.com/gluster/gluster-prometheus/gluster-exporter/metric_volume_counts.go:144 +0xb2c
main.main.func1(0xc00018e250, 0xc000023140, 0x15, 0x5, 0x0, 0x9edab8, 0x15, 0xa0ab90, 0xa8d920, 0xc000063940)
	/opt/gluster-exporter/src/github.com/gluster/gluster-prometheus/gluster-exporter/main.go:116 +0xb9
created by main.main
	/opt/gluster-exporter/src/github.com/gluster/gluster-prometheus/gluster-exporter/main.go:112 +0x5a3
```

--- a/gluster-exporter/metric_volume_counts.go
+++ b/gluster-exporter/metric_volume_counts.go
@@ -17,6 +17,10 @@ var (
 		},
 	}
 
+	countLabels = []MetricLabel{
+		clusterIDLabel,
+	}
+
 	volumeCountGaugeVecs []*prometheus.GaugeVec
 
 	glusterVolumeTotalCount = newPrometheusGaugeVec(Metric{
@@ -24,7 +28,7 @@ var (
 		Name:      "volume_total_count",
 		Help:      "Total no of volumes",
 		LongHelp:  "",
-		Labels:    []MetricLabel{},
+		Labels:    countLabels,
 	}, &volumeCountGaugeVecs)
 
 	glusterVolumeCreatedCount = newPrometheusGaugeVec(Metric{
@@ -32,7 +36,7 @@ var (
 		Name:      "volume_created_count",
 		Help:      "Freshly created no of volumes",
 		LongHelp:  "",
-		Labels:    []MetricLabel{},
+		Labels:    countLabels,
 	}, &volumeCountGaugeVecs)
 
 	glusterVolumeStartedCount = newPrometheusGaugeVec(Metric{
@@ -40,7 +44,7 @@ var (
 		Name:      "volume_started_count",
 		Help:      "Total no of started volumes",
 		LongHelp:  "",
-		Labels:    []MetricLabel{},
+		Labels:    countLabels,
 	}, &volumeCountGaugeVecs)
 
 	glusterVolumeBrickCount = newPrometheusGaugeVec(Metric{

